cnd/src/network/tor.rs

```diff
@@ -88,12 +88,6 @@ impl TorTokioTcpConfig {
         }
     }
 
-    /// Sets the TTL to set for opened sockets.
-    pub fn ttl(mut self, value: u32) -> Self {
-        self.ttl = Some(value);
-        self
-    }
-
```

Comment:
Interesting PR, I was actually thinking about this dead code issue recently when I added the `pub` to this module so that this very function didn't throw a clippy warning :)